src/services/bleu/metadata/types.ts

```diff
@@ -0,0 +1,14 @@
+export type Address = string;
+export type QueryArgs = Record<string, any>;
+export type QueryAttrs = Record<string, any>;
```

Comment:
what is this needed for? Is this a pattern they already use in their repo for gql queries?

Reply:
Not really sure for what is needed, but I tried to follow the gauge implementation since it's an extra subgraph just like ours https://github.com/bleu-studio/balancer-frontend-v2/blob/develop/src/services/balancer/gauges/types.ts